generators/kubernetes/templates/cert-manager/letsencrypt-staging-ca-secret.yml.ejs

```diff
@@ -0,0 +1,11 @@
K1bdOp7nbzm2umKraNMlSJ6kal863WiFeJMYKpp575ff52QCRiR4ZR8o+my0pqxnq8XAzu5HNtgO3KxgtiwQxufywODY3JgBA+7SPX3CAusQW4Sm9yRIqS2IpsdnTtoUduo/xybalkfKb6aDpJEWhJXTekyP6+7cZaKihj2FFJlPivrZ0VIwqu+z+IBjnX9KBWBz5xs4JW8yPuo3vwU8Ohgh7PcWayAnKPNrm2B3U6YlsjnSNtfbvjTcGv1BpKuirDQCb8aiq2od+GpGFilD4MmIcyq2OxB/FFinqguYSiIMNGkvffyU2vzwUv02xUmwqODg+FuJ4iwjLmNQmGLHKfx/QBwdQJKK3kPlW/D4CyiyxKrXQ9O3JkI4cBKrHk4xKZF9xktNk1xoBVPHehmO7RsYjOjJP6UYaVHOPgTNyXp4DVMee/3lKyNdZDwp9syux7izJ02CFNVaoh0eBxrAj9rCGUtojyyibuADYUQS9mPL//LzMntNtm0YvMJADGc3LUMwMXTBgTv7nTSomKKXws8wMwWp3XRMQTo+tsV8P3gPKM+G5maOMSZCQ2BEi94pgs9d2PtlYC1is9uvY+4RS7ouoOBvvH3+Pr7S2arKe2ZsMsbdLuZMweho56gYG2dPVnknobdKl+8axWp3VsPZmyTuQfyOci7rSUXPw+dHLg6WyHibETuAyHmkt+eLXPt9ofgIqa/apH1sc8Wv93TeoJoCIG8uk1OOCxDyb7Fjbt+EVkxRC9TOx+4NJpkWLRvcpBjGbN2wMTpucL9ZuRk6zZx6GOXEE8KIoLhV74RQ8HP22fuv00kjH8QBoetqyslFLWl3rmixITp9JzWdeqxO7TgrdnOtZ30YgSsbOy63xOXAtN/fDFUvNyOsD/QBj5S9oCu0oAym/plZ4A9GLp340Mo1AV6navk1ECK3Fm9rtvqPv/qpVsJ2JDoeSfPmPpn37+VcP7pIa8LqecAszapW3W6mDfNQhX1M0xjHOiRhXra1xP9gApG/z9WfkWvl34icWz635fb0mr+mg6KZRXOR2mOt5IpwTqacP+Vrr3cIkqWm7gnQk6tU5wHFGI0tyKHPuMz9rdxQwN2HB4+BH5T1//FNcA5qLovRSln22+Sh9oVRmj/dQwAY4ENtF2cEZxC7uum3BgeEi/vPZVHDPcuD7qeYmC/Xt/b4ELavJGues2pbOagXrf+dTDbJAinmGuf6N1cBeWtUBX0Q3R1Ndct2xJaOG9IMpqqKimK7AiSPGVg7cjvTjrapHmFlJBeWzoDqBLWSo/zGiIDBnx17mNd5f8oS+GYzAkO1AWiY1DYheh4cTPAxvv5KB7JoFw55XfA3zyiDZBKzmQOr648c9/ERma1HmQ9SjPGg2FMnwOpRMQL6FgIwavKPpqReq8GxVJRURIUf8XJhzvqrqv7PO1ZQUHs4egYaNClcbWQ4/dvqAM6uJ8npCpAlyEKRAhi5T7buYNu2olie1ZMulBSQKbpk8O/bX/EI9P8tJwSYvAHBch+hHhAQATZUI3yJ/4KQpW9AYG50FcI5ZGVEoIZ6FtRri2yUjGuPw//YA8tFS2WbDiv3gk7O6kFko+50sqiYYLJ1dwj3DljvwlxXsKercZ7VXD6mvOQjQA1K4bH2lG3SID2BtkT4JeYu5eWBdEudw63+TdbwT+7jTKbWGEFhCSaTmjNjUXm7QWaIdJdRDBkCSAjWG1s9sHHRyuf4mNHf+jSkjRepUMfr24WJdW0+FDKsMO7f5g4moJwPL4aUgmYm/CBPlZmn51BvE7GCStRsj5PGdcmHkjF561CyLt3hhmLRGd1dTIW6mEoJDCvenHPf/OfNmDC29aM7BE0yGqsBCTF8ORhTqSC2C2I+il9fNl2/bpf9m/BeOAwI0MFo6RAfPYoPdl2gONlVZxRzjy83NAc+FiERv3WBAfe6iEWWsjKV8iV1usJ35V9FU12mqE5Fzd2ix0oVIjevo9e6l+lLXwci9mKi9sI9WXG/97W/7uuYaZMIGlYY0aLONgkqveBMW04T/g89P6ghVJOhD/bb5fLDgVYf8hIXUh5nBNQEJhK8Us+pW+cOSeQpwReV/a6D9/ImvpoQcFK1wYCMmAPGFw7RyakSoBhfwMvINhwQdSB/45GfXymTUjf5eTfcLEmUgdSmPv6vgQJu2ClyQraapt6AYKTM+ddGWcw/U3HRSK+ptZVXBmmecbvCnSTlF2mZuL3TXCYN4DBASZ0pSLGpCkOrDuhvM2M9YW/bK0z28YZWVAF5hObA6uTxVCvnDIsDukXqQNDdN5eyU2MaWAXVunw50YdQihNQEf3HGWY2Rm/iMAuh6pNg99dXDZLXWtgCKoXl/lH9IOO7cg2uz2yrAWXkujRpPVW6CuIM2pcl0H6s5i7/mUW5rdgEX5rIUQLYVDCiADuWM3UUjBCgcaOM4EDTCMYRI0ELrIbHU3FKClDeeLdr7hMBPassWMlZFfO/bHQg/YrwdD0bcmEDjXPIprrA/fcb9lHU7wGxVqw3WVRdggw62ufmFbcyMZhRSUfQcbOcSS9bI1G/2m5XYmIe6zrSjv7n2p0ExPVD/FnkcU4b3OxzvVRZYyz5b+1fAuxl2LWA0CTgwb7h+iTA0UWnmZT2SsQdOb16ZgZdpJJcYcpqLlBabCuUu+HfehY9gczqH5+ET6jFHKOgxbf6PNyXE2y1JA2v2A2eIZ3gmEMsfZ0bHkWvtxmw+qp93bMa137jU7vrXdXNaXLAAAjWRr6G1IaSpsnQVGLAfIH8NlmNkW1yZ2HZ2SajDcYzdsC3MBYBfEY/K5grGZMT7PBu3KVCV0SxTvUaXLaSSMysNBRN1IcJ96er+EivYFamVrhfR8D2gsYuiax3lWIwjTeE3o3O5eJJJdhgT8Jk+n8AZbQwTTAxMA0GCWCGSAFlAwQCAQUABCBY3bHkvPayRA0gqLCPhPJxw4jgA2kLISJtqzFZfHqy/gQUNElj1rRfftSsbdj7vrKhdp7I6I8CAicQ
```

Comment:
Is this a public or private key?

Reply:
Public keys. 
The data in truststore.jks is a manually generated truststore, which contains Let's Encrypt staging intermediate and root certificates (public keys). The truststore is required because the Keycloak certificate for TLS, issued by staging, is not trusted by the client applications (registry, microservices) and therefore the apps won't start. The truststore is protected by a password that right now is in plain text in the deployment.yml.ejs template.